src/Toml/Codec/Generic.hs

```diff
@@ -712,7 +711,6 @@ instance (Generic a, GenericCodec (Rep a)) => HasCodec (TomlTable a) where
 
 instance (Generic a, GenericCodec (Rep a)) => HasItemCodec (TomlTable a) where
     hasItemCodec = Right $ Toml.diwrap $ genericCodec @a
```

Comment:
Let's add `INLINE` pragmas for this codec to have slightly better chances of having slightly better performance...